v03_pipeline/lib/annotations/enums.py

```diff
@@ -204,7 +204,7 @@
     'Likely_pathogenic/Likely_risk_allele',
     'Established_risk_allele',
     'Likely_risk_allele',
-    'Conflicting_interpretations_of_pathogenicity',
+    'Conflicting_classifications_of_pathogenicity',
```

Comment:
excellent point

Reply:
do we need to support the possibility of both values being in clinvar reference data? this will happen if we update the MITO clinvar dataset before SNV_INDEL (which is likely)

Reply:
Yeah, I think we should probably try to be backwards compatible.  I'd ping Hana on this one so she expects the PR and can give guidance on whether or not it's possible.

Reply:
The other mito dataset that was flagged to be updated is `dbnsfp_mito` (which makes sense because the selects and enums and path are different in `config.py` from what's in the rdc table). One seqr change I see here is switching [SIFT_pred](https://github.com/broadinstitute/seqr/blob/760e8c80d1164b2b3b14c326cd140d788d8a37e0/hail_search/queries/mito.py#L49) to [SIFT_score](https://github.com/broadinstitute/seqr-loading-pipelines/blob/dev/v03_pipeline/lib/reference_data/config.py#L80)

Reply:
Yeah, I think that also hasn't been updated yet!

Reply:
So its technically possible, because the mito and snv indel  code are distinct, currently they use a shared constant but it would be possible to override the constant and have them use different constants. However, given that we seem to have a few mito changes on deck and the mito table is not that big, it might just be better to reannotate the mito table with the latest data even though we have no data to reload, and that way we don't need to worry about this in the future when we do get around to updating it